src/ConfigDiscovery.php

```diff
@@ -6,6 +6,9 @@
 
 namespace Zend\ComponentInstaller;
 
+use Zend\ComponentInstaller\ConfigDiscovery\DiscoveryChain;
+use Zend\ComponentInstaller\Injector\ConfigInjectorChain;
```

Comment:
These don't need to be imported, and, in fact, other classes in their respective subnamespaces are not; remove for consistency, please. :smile: